deploy/docs/Best_Practices.md

```diff
@@ -1030,6 +1031,90 @@ metadata:
         value: '12345'
 ```
 
+## Using newer Kube Prometheus Stack
+
+Due to breaking changes, we do not support the latest Kube Prometheus Stack.
+We are aware that it can be a major issue, so this section describes how to install Kube Prometheus Stack in version `34.10.0`
+to work with our collection.
+
+1. Prepare [values-prometheus.yaml] with Kube Prometheus Stack configuration:
+
+   - copy content of `kube-prometheus-stack` from [values.yaml]
+   - add the following configuration:
+
+     ```yaml
+     prometheus:
+       prometheusSpec:
+         initContainers:
+           - name: "init-config-reloader"
+             env:
+               - name: FLUENTD_METRICS_SVC
+                 valueFrom:
+                   configMapKeyRef:
+                     name: sumologic-configmap
+                     key: fluentdMetrics
+               - name: NAMESPACE
+                 valueFrom:
+                   configMapKeyRef:
+                     name: sumologic-configmap
+                     key: fluentdNamespace
+     ```
+  
+   - remove the following configuration:
+
+     ```yaml
+     prometheus:
+       prometheusSpec:
+         thanos:
+           ## Use our own image until Thanos upstream adds arm64 docker images
+           ## to their registry.
+           baseImage: public.ecr.aws/sumologic/thanos
+           version: v0.23.1
+     kube-state-metrics:
+       ## Use the GCR repo, it's more recent and has ARM images starting from 1.9.8
+       image:
+         repository: k8s.gcr.io/kube-state-metrics/kube-state-metrics
+         tag: v1.9.8
+     prometheus-node-exporter:
+       image:
+         tag: v1.3.1
+     ```
+
```

Comment:
Customer should add also custom changes from existing configuration for sumologic helm chart, e.g. when additional metric was in remote write configuration etc.